src/Common/HashTable/Hash.h

```diff
@@ -389,6 +391,22 @@ struct UInt128HashCRC32 : public UInt128Hash {};
 
 #endif
 
+struct UInt128HashNew
```

Comment:
CRC hash will still be fine unless you need more than 32 bits of entropy. I would not expect any savings compared to 2 consecutive CRCs in clickhouse

Reply:
yeah, I didn't expect much, but was interesting to try anyway